src/types/BalanceDelta.sol

```diff
@@ -22,7 +22,13 @@ function sub(BalanceDelta a, BalanceDelta b) pure returns (BalanceDelta) {
     return toBalanceDelta(a.amount0() - b.amount0(), a.amount1() - b.amount1());
 }
 
+function eq(BalanceDelta a, BalanceDelta b) pure returns (bool) {
+    return a.amount0() == b.amount0() && a.amount1() == b.amount1();
+}
+
 library BalanceDeltaLibrary {
+    BalanceDelta public constant MAXIMUM_DELTA = BalanceDelta.wrap(int256(type(uint256).max));
```

Comment:
I just did it because Mark requested we use a sentinel value [here](https://github.com/Uniswap/v4-core/pull/324#discussion_r1397789015) - and 0 isnt a sentinel value really in this situation. Will add a comment!